app/latigo/metadata_api/metadata_storage_provider.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from typing import Dict, Tuple
+
+from latigo.metadata_api.client import MetadataAPIClient
+from latigo.metadata_api.data_structures import InputTag, OutputTag, TimeSeriesIdMetadata
+from latigo.metadata_storage import MetadataStorageProviderInterface
+from latigo.time_series_api.misc import make_output_tag_derived_from, make_output_tag_description, make_output_tag_type, \
+    make_prediction_metadata_description
+from latigo.types import PredictionDataSet
+
+logger = logging.getLogger(__name__)
+
+
+class MetadataAPIMetadataStorageProvider(MetadataAPIClient, MetadataStorageProviderInterface):
+    def __init__(self, config: dict):
+        super().__init__(config)
+
+    def __str__(self):
+        return f"{self.__class__.__name__}({self.base_url})"
```

Comment:
This representation looks misleading, since `MetadataAPIMetadataStorageProvider(base_url)` would raise an error. Use angular braces to indicate that string representation can't be used to initialize the class.
```suggestion
        return f"<{ type(self).__name__ }: { self.base_url }>"
```